exporters/mongodb3/mongodb3.prometheus.json.tmpl

```diff
@@ -7,8 +7,8 @@
     "integrations": [
          {
             "name": "nri-prometheus",
-            {{- if .env.interval }}
-            "interval": "{{.env.interval}}",
+            {{- if .env.NRI_CONFIG_INTERVAL }}
```

Comment:
Would it make sense to explain in a comment how the `NRI_CONFIG_INTERVAL` env variable is set?

Reply:
I thought about it but i don't think I could add a comment here without being templated. 
I added the comment on the tests if you check

Reply:
I don't now if this syntax:

```
{{/* Comment */}}
```

Would work or would be problematic. It is not important, anyway.

Reply:
yea not great IMO , I think that a quick ctrl+f would give the hint to anyone trying to dig into it really quick and see the test comment.